geode-core/src/test/java/org/apache/geode/management/internal/configuration/realizers/RegionConfigRealizerTest.java

```diff
@@ -33,12 +34,14 @@
 public class RegionConfigRealizerTest {
   InternalCache cache;
   RegionFactory regionFactory;
+  RegionConfigRealizer realizer;
 
   @Before
   public void setup() {
     cache = mock(InternalCache.class);
     regionFactory = mock(RegionFactory.class);
     when(cache.createRegionFactory()).thenReturn(regionFactory);
+    realizer = spy(RegionConfigRealizer.class);
```

Comment:
This does not need to be a spy, just instantiate a `new RegionConfigRealzer()`

Reply:
agree

Reply:
fixed in latest commit.